### ISSUE TITLE
Cannot set columns to be stacked in navigator

### ISSUE THREAD

**Issue by kamilkulig** [highcharts/highcharts#7033]
http://jsfiddle.net/kkulig/x1a2ggq1/

#### Expected behaviour
Following code: 
```
navigator: {
    series: {
       stacking: 'percent'
    	type: 'column'
    }
  }
```
should set columns to be stacked in navigator.

#### Actual behaviour
No series are displayed in navigator.

**Comment by pawelfus** [comment]
Partially fixed in v6.0.4: http://jsfiddle.net/BlackLabel/x1a2ggq1/4/ - animation is still a bit off

**Comment by sebastianbochan** [comment]
It looks like fixed since 7.0.3.

http://jsfiddle.net/BlackLabel/gz5n27y4/

Please reopen when problem will appear.
